students/sikorski_marcin/Lesson_3_Python_basics.py

```diff
@@ -0,0 +1,158 @@
+#The length of segment
```

Comment:
Jako, że wkroczyliśmy w świat funkcji, które umożliwiają nam tworzenie modułów i projektów podzielonych na kilka plików/modułów, podziel poszczególne zadania na osobne moduły (pliki) z nazwą nazwa_zadania. Możesz je trzymać w osobnych folderach  np: lesson_03_functions.
Aktualny sposób organizacji kodu sprawia, że masz wymieszane funkcje z luźnym kodem, który jest poza funkcjami.
Efekt jest taki, że jak zaimportujesz ten moduł/plik do innego skryptu bądź modułu to podczas importu zostanie wykonany kod, który jest 'luzem'. Aby sie przed tym zabezpieczyć używa się konstrukcji:
`if __name__ == '__main__':`
Dzięki czemu podczas importu z modułu do innego modułu/skryptu kod, który jest wewnątrz tego bloku `if` nie zostanie wykonany.
On się wykona tylko wtedy, gdy ten moduł/skrypt zostanie uruchomiony jak skrypt (na najwyższym poziomie).
Jeżeli uruchomisz ten skrypt zmienna name przyjmie wartość 'main', jeżeli zaimportujesz jako moduł to name ma wartość: 'nazwa_pliku'.